test/file_test.rb

```diff
@@ -503,52 +514,58 @@ def test_commit_use_zip_entry
   #  end
 
   def test_compound1
-    renamedName = 'renamedName'
+    renamed_name = 'renamed_name'
```

Comment:
Technically would still work with the old value, I think.

Reply:
Yes I could have left the value unchanged, but was being competitionist about things :-)